eukarya-inc/PLATEAU-VIEW-3.0
# feat(extension): timeline field components

## Overview

This PR adds these field components:
- General: Apply time value field
- General: Timeline cutomized field
- General: Timeline month field
- Point: Convert from csv field

## Screenshot
![image](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/21994748/a32d5c74-10be-4031-8758-e102cb1aea06)
![image](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/21994748/8516d77d-4127-4865-bb5a-20d23b5c0916)


## Test
- Use dataset `USGS(米国地質調査所)地震ハザードリアルタイム通知、フィードウェブサービス情報` to test `Apply time value field` and `Convert from csv field`. Note: need to zoom out to check points in global. Turn on/off `Time based display` to show limited / all points.
- Use dataset `東日本大震災時の帰宅困難者（東京駅周辺）` to test `Timeline cutomized field`
